protocol/protocol.go

```diff
@@ -438,6 +438,14 @@ type GroupMessage interface {
 	Group() string
 }
 
+// TransactionalMessage is an extension of the Message interface implemented by some
+// request types to inform the program that they should be reouted to a transaction
```

Comment:
typo: reouted  -> routed (I think!)

Reply:
Sorry, I think I merged just before your comment. I'll get that typo fixed up in another PR. Thanks!